renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -0,0 +1,604 @@
+#include <stdio.h>
+#include <sys/types.h>
```

Comment:
I'm not sure `sys/types.h` exists portably on windows. What did you need it for, is there a portable alternative or one already available in `os_specific.h`? Similarly what is stdio needed for here?

The file is also missing the copyright comment header.

Reply:
Sorry, I should have removed stdio, for sys/types.h I am not sure why I put it there, I had a lot more code there that I deleted, so it might have been left from that. I removed it now.
Thanks for catching this!